Documentation/Contributors/BuildGuide/README.md

```diff
@@ -44,9 +47,26 @@ Cesium ships with a simple HTTP server for testing, run `npm start` after buildi
 
 ```
 npm start
-```
+``` 
+
+Then browse to [http://localhost:8080/](http://localhost:8080/). You should see the following:
+
+* **Apps**
+   * **Cesium Viewer** : our core 3D globe viewer with some basic features included
+   * **Hello World** : basic app to [get you started](http://cesiumjs.org/tutorials/cesium-up-and-running/)
+   * **Sandcastle** : live coding app for viewing and modifying source code from [many examples](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Hello%20World.html&label=Showcases)
+   * **Cesium Inspector** : Cesium Viewer with extra view options to expose underlying technical details
+   * **Timeline Demo** : demo of the Cesium Timeline widget, for use with time-dynamic data
+* **Test Suites**
+* **Documentation**
 
-Then browse to [http://localhost:8080/](http://localhost:8080/).
+These apps have slightly different build processes. The basics:
+
+* `npm run build` will build Cesium Viewer, Sandcastle, Cesium Inspector and Timeline Demo from source. 
+* `npm run combine release` will build Hello World, which uses a built version of Cesium.
+* `npm run generateDocumentation` will [build the docs](https://github.com/rahwang/cesium/tree/master/Documentation/Contributors/DocumentationGuide).
```

Comment:
Link here: http://cesiumjs.org/refdoc.html

`build the docs` -> `build the documentation`

Avoid abbreviations where possible because they can be confusing to those who aren't native English speakers